app/task_detail.go

```diff
@@ -328,6 +328,15 @@ func (td *TaskDetailPane) handleShortcuts(event *tcell.EventKey) *tcell.EventKey
 		case 'x':
 			td.Export()
 			return nil
+		case 'o':
+			td.setTaskDate(parseDateInputOrCurrent("").Unix(), true)
+			return nil
+		case '+':
+			td.setTaskDate(parseDateInputOrCurrent(td.taskDate.GetText()).AddDate(0, 0, 1).Unix(), true)
+			return nil
+		case '-':
+			td.setTaskDate(parseDateInputOrCurrent(td.taskDate.GetText()).AddDate(0, 0, -1).Unix(), true)
```

Comment:
Just a quick idea, can we reuse previously defined `todaySelector()`, `prevDaySelector()`, `nextDaySelector()` methods here?

Reply:
I have no idea to be honest, my experience with Go pretty much non existent. But I guess it works, otherwise you probably wouldn't have suggested it :wink:

I'll check out the tutorials and see if I can figure it out :slightly_smiling_face:

Reply:
Maybe I can assist. You need to do a few changes:

1. Remove `todaySelector()`, `prevDaySelector()`, `nextDaySelector()` functions from inside `makeDateRow()`
<img width="1067" alt="CleanShot 2021-10-06 at 23 41 19@2x" src="https://user-images.githubusercontent.com/439612/136255711-d7e08f68-797a-4323-9903-5156a279feae.png">

2. Now, at the end of file, define them as methods of `TaskDetailPane`
<img width="961" alt="CleanShot 2021-10-06 at 23 43 47@2x" src="https://user-images.githubusercontent.com/439612/136255942-12f38b61-25ce-48f7-ba56-cf7b7646ed94.png">

3. In`makeDateRow()`, use these methods
<img width="671" alt="CleanShot 2021-10-06 at 23 45 18@2x" src="https://user-images.githubusercontent.com/439612/136256153-e48172a7-8e24-455a-99cb-0c4a4a131538.png">

4. Use the same methods in shortcut switch-case
<img width="483" alt="CleanShot 2021-10-06 at 23 45 48@2x" src="https://user-images.githubusercontent.com/439612/136256269-94ce2cce-892c-4d2a-a2bf-5793a39ba736.png">

Then compile and test, hope it'll work. :)
The only thing we're achieving by these changes is - avoiding duplication of the same logic in several places.